golang/README.md

```diff
@@ -8,11 +8,10 @@
 ## Test Command
 
 ```sh
-$ ginkgo
+$ go test .
```

Comment:
Perfect! The reason I used `ginkgo` here was ignorance about `go test -v .`: 

The test reporter needs to be able to tell me "1 test passed, 1 test failed" in a way that can be parsed (see below for an example).

https://github.com/rradczewski/kata-bootstraps/blob/0248ec631bd50b55feec358afd34bfcaf59180ef/golang/.devcontainer.json#L7-L8

I think `go test -v` should have the same output as `ginkgo` does, so I'm happy to get rid of `ginkgo`.

The README is furthermore generated from the `.devcontainer.json`, so your changes would have to go there to take effect.